app/src/test/java/com/client/shop/ui/account/contract/ChangePasswordPresenterTest.kt

```diff
@@ -117,8 +113,8 @@ class ChangePasswordPresenterTest {
     fun shouldShowPasswordMatchErrorOnDifferentPass() {
         presenter.changePassword("12345678", "123456789")
         verify(view).passwordsMatchError()
-        verify(useCase, never()).execute(any(), any(), eq("12345678"))
-        verify(useCase, never()).execute(any(), any(), eq("123456789"))
+        verify(useCase, never()).execute(any(), any(), any())
```

Comment:
Do we need to verify this twice?

Reply:
no, it is a duplicate code, will delete it